weechat-rs/src/buffer.rs

```diff
@@ -300,6 +300,26 @@ impl Buffer {
         }
     }
 
+    /// Display a message on the buffer with attached date and tags
+    pub fn print_tags_dated(&self, date: i64, tags: &str, message: &str) {
```

Comment:
We might want to support some higher level date type here. For example the `DateTime` type from [Chrono](https://docs.rs/chrono/0.4.7/chrono/struct.DateTime.html)

The `DateTime` type can be trivially converted to an u64 unix timestamp.

Maybe we should support both, since Matrix provides unix timestamps as the message date and this would avoid conversions. I assume Discord does the same?

Anyways, this is fine to merge and we can provide support for a higher level time type later on.

Reply:
That's a good point, Discord provides unix timestamps so I never thought about using chrono